apps/minifront/src/components/extension-not-connected.tsx

```diff
@@ -14,26 +14,33 @@ export const ExtensionNotConnected = () => {
 
   useEffect(() => {
     if (approved === true) location.reload();
+    else document.title = 'Penumbra Minifront';
```

Comment:
This feels like an odd place to put metadata. Think it's better we use something like react-helmet or something. Perhaps out of scope for this PR though.